string node_id
string robot_id
